### PR TITLE
Add mmdev command with i18n check and extract

### PR DESCRIPTION
**Pull request by jespino**
#### Summary
Add mmdev command with i18n check and extract added and removed translations

### CODE REVIEW

**Comment by cpanato** [comment]
I think those commands should go to another repo. this don't need the server to run, you can pass the folder to the command.

**Comment by jespino** [comment]
Yes, I agree, It is easy to move to another repo, and allow us to include other stuff in this tool. Thinks like `mmdev sign-binary --cert whatever.pem mattermost` or `mmdev license-header` to write correctly the license in all the go files. maybe something like `mmdev check` with our own checks about the code, for example, no more GetReplica with INSERT, DELETE or UPDATE. I'll leave it here until is reviewed, but after that I can create a mattermost/mmdev and create it there.

File: cmd/mmdev/commands/i18n.go [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+)
+
+type Translation struct {
+	Id          string      `json:"id"`
+	Translation interface{} `json:"translation"`
+}
+
+var I18nCmd = &cobra.Command{
+	Use:   "i18n",
+	Short: "Management of mattermost translations",
```
Comment (jwilander) [inline cmd/mmdev/commands/i18n.go]: `mattermost` -> `Mattermost` :P

**Comment by crspeller** [comment]
This is going in an external repository.

**Pull request closed without merging**